code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/hedgehog.dm

```diff
@@ -103,6 +102,15 @@
 	var/percentage_shards = round((shards / max_shards) * 100, 10)
 	if(percentage_shards)
 		holder.overlays += image('icons/mob/hud/hud.dmi', "xenoenergy[percentage_shards]")
+
+	if(percentage_shards >= 50)
+		bound_xeno.small_explosives_stun = FALSE
+		bound_xeno.remove_filter("hedge_unstunnable", 1, list("type" = "outline", "color" = "#421313", "size" = 1))
+	if(percentage_shards < 50)
+		bound_xeno.small_explosives_stun = TRUE
+		bound_xeno.add_filter("hedge_unstunnable", 1, list("type" = "outline", "color" = "#421313", "size" = 1))
```

Comment:
Don't we generally apply outlines when some ability is active? I think we should probably reverse this where if you are unstunnable you have an outline, otherwise you are normal.